src/components/stack-chart/index.js

```diff
@@ -166,7 +167,11 @@ class StackChartImpl extends React.PureComponent<Props> {
     }
 
     if (event.key === 'Enter') {
-      const bottomBoxInfo = callTree.getBottomBoxInfoForCallNode(nodeIndex);
+      const bottomBoxInfo = getBottomBoxInfoForCallNode(
+        nodeIndex,
+        callNodeInfo,
+        thread
```

Comment:
Just a note that `thread` here is the "FilteredThread" from `getFilteredThread` but the call tree uses the thread from `getPreviewFilteredThread`.
I think that's fine because `getBottomBoxInfoForCallNode` doesn't use the sample data, but wanted to point it out in case you find some other issue.

Reply:
That's a good point, I hadn't considered this. You're right that it doesn't make a difference today. However, maybe `getBottomBoxInfoForCallNode`, or more specifically `getNativeSymbolsForCallNode`, *should* be using sample data: When we use the assembly view on a JITted JS function, and this function has been compiled multiple times into different native symbols, it would be nice to only show the native symbols / "compiled versions" that are in the current view.

I'm not going to worry about this use case now, but I'll keep it in mind for the future.